src/components/SendActivityModal/hooks/useSendActivityModal.ts

```
@@ -111,7 +121,8 @@ export const useSendActivityModalStore = create<SendActivityModalStore>(
         formDataToValidate = { ...state.formData, students };
         updatedFormData = formDataToValidate;
       }
-      const errors = validateStep(state.currentStep, formDataToValidate);
+      const options = { enableProvaMode: state.enableProvaMode };
+      const errors = validateStep(state.currentStep, formDataToValidate, options);
```

ESLint

[error] Replace `state.currentStep,·formDataToValidate,·options` with `⏎········state.currentStep,⏎········formDataToValidate,⏎········options⏎······`

(prettier/prettier)